docs/src/@primer/gatsby-theme-doctocat/nav.yml

```diff
@@ -111,6 +113,8 @@
       url: /Text
     - title: TextInput
       url: /TextInput
+    - title: TextInputField
+      url: /TextInputField
```

Comment:
Should this be `InputField` now?
![Screenshot 2021-12-15 at 15 08 53](https://user-images.githubusercontent.com/13340707/146212004-3ff0f340-8169-4f2a-bf54-f14b583a0f1c.png)

Reply:
Ooops my mistake 🙈